src/NzbDrone.Core/Authentication/UserService.cs

```diff
@@ -56,13 +70,32 @@ public User Update(User user)
             return _repo.Update(user);
         }
 
+        // TODO: Do not allow to change username to something that already exists.
```

Comment:
This can be done in API validation, seem how it's done for indexers/custom formats/etc

Reply:
Just realized this was already implemented by default just tested it now, getting this error which is expected `constraint failed
UNIQUE constraint failed: Users.Username`.

Would most likely need to implement frontend change to show this error.

Marking this as resolved for now.

Reply:
The validations till needs to be done in the API to return an error message to the UI, relying on on the DB unique check will result in a 500 response instead of 400.

Reply:
Ah right sure, sounds good thanks @markus101